app/src/ui/about/about.tsx

```diff
@@ -232,7 +237,7 @@ export class About extends React.Component<IAboutProps, IAboutState> {
           </Row>
           <h2>{name}</h2>
           <p className='no-padding'>
-            Version {version} ({releaseNotesLink})
+            <LinkButton title='Click to copy' className='version-text' onClick={this.onClickVersion}>Version {version}</LinkButton> ({releaseNotesLink})
```

Comment:
I read through #1778, and it’s not clear what you’d like me to that's from that. Should I have a separate “click to copy” link?